lib/src/data/data_manager.dart

```diff
@@ -740,50 +740,52 @@ class DataManager {
       }
     }
 
-    for (final String layoutID in variableUpdates.keys) {
-      final UpdateType updateType = variableUpdates[layoutID]!;
+    // docID is can be either a layoutID or a canvasID.
+    // layoutID is for backward compatibility.
+    for (final String docID in variableUpdates.keys) {
```

Comment:
This docID refactor is extremely confusing. I'm not sure what is being targetted anymore. Please clarify it more in these docs.